server/src/test/java/org/elasticsearch/env/NodeMetadataTests.java

```diff
@@ -106,16 +112,32 @@ public void testDoesNotUpgradeAncientVersion() {
         );
         assertThat(
```

Comment:
For this test we should adjust the definition of `tooOldVersion` so that it uses `minimumCompatibilityVersion` instead of `minimumIndexCompatibilityVersion`.